CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Changelog
 
-## 4.2.1 (not published)
+## version tbd (not published)
 
 ### Changed
 
 -   Updated styles for `<DropdownToolbar>` to support Icon Button or plain icon for `navigationIcon`.
+-   Updated default styles for `<Drawer>`
 
-### Fixed
-
--   `<Drawer>` nested nav item dark-themed background color.
+### Added
+-   Additional configuration properties for `<Drawer>`
+    -   `sideBorder` gives the `<Drawer>` a slight border instead of a drop shadow
+    -   `disableActiveItemParentStyle` disables the bold text style for active item's parent elements
+-   Additional configuration options for `<DrawerHeader>` and `<DrawerFooter>`
```

Comment:
`hideContentOnCollapse` was added to `<DrawerSubHeader>` not `<DrawerHeader>`, right?